daml-lf/validation/src/test/scala/com/digitalasset/daml/lf/validation/TypingSpec.scala

```diff
@@ -744,15 +759,25 @@ class TypingSpec extends AnyWordSpec with TableDrivenPropertyChecks with Matcher
           },
         E"Λ (σ : ⋆). λ (e: σ) → ⸨ create @Mod:T e ⸩" -> //
           { case _: ETypeMismatch => },
+        // UpdCreateInterface
+        E"λ (e: Mod:U) → ⸨ create_by_interface @Mod:U nothing ⸩" -> //
+          {
+            case EUnknownDefinition(
+                  _,
+                  LookupError(Reference.Interface(_), Reference.Interface(_)),
+                ) =>
+          },
+        E"Λ (σ : ⋆). λ (e: σ) → ⸨ create_by_interface @Mod:I e ⸩" -> //
+          { case _: ETypeMismatch => }, */
         // UpdExercise
-        E"λ (e₂: List Party) (e₃: Int64) → ⸨ exercise @Mod:U Ch nothing e₂ e₃ ⸩" -> //
+        E"λ (e₁: ContractId Mod:U) (e₂: List Party) (e₃: Int64) → ⸨ exercise @Mod:U Ch e₁ e₂ e₃ ⸩" -> //
```

Comment:
I’m confused, isn’t there one argument too much here? It seems like exercise should only have the cid & choice arg but we have e1, e2, e3

Reply:
Good catch.